contracts/bridge.sol

```diff
@@ -152,6 +159,7 @@ contract HomeBridge {
     }
 }
 
+
```

Comment:
Seems redundant.

Reply:
it's a default solium rule: http://solium.readthedocs.io/en/latest/user-guide.html#list-of-core-rules
search for rule `blank-lines`: `Ensure that there is exactly a 2-line gap between Contract and Function declarations`

i think it increases readability a bit

Reply:
I see, fair enough.